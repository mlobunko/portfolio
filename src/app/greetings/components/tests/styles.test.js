import React from "react";
import { shallow } from "enzyme";
import "enzyme-to-json";

import {
  StyledArrow,
  StyledBottom,
  StyledBottomText,
  StyledGreetings,
  StyledHeader,
  StyledHeaderFirstSentence,
  StyledHeaderSecondSentence
} from "../styles";

describe("StyledArrow", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledArrow />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledBottom", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledBottom />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledBottomText", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledBottomText />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledGreetings", () => {
  const props = {
    image: "123"
  };
  it("renders correctly", () => {
    const wrapper = shallow(<StyledGreetings {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledHeader", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledHeaderFirstSentence", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledHeaderFirstSentence />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledHeaderSecondSentence", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledHeaderSecondSentence />);
    expect(wrapper).toMatchSnapshot();
  });
});
