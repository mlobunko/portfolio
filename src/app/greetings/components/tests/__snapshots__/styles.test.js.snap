// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledArrow renders correctly 1`] = `
<div
  className="sc-bdVaJa griMVU"
/>
`;

exports[`StyledBottom renders correctly 1`] = `
<div
  className="sc-bwzfXH hBpGIZ"
/>
`;

exports[`StyledBottomText renders correctly 1`] = `
<div
  className="sc-htpNat gOUWiF"
/>
`;

exports[`StyledGreetings renders correctly 1`] = `
<div
  className="sc-bxivhb kQojpz"
/>
`;

exports[`StyledHeader renders correctly 1`] = `
<div
  className="sc-ifAKCX itfMJV"
/>
`;

exports[`StyledHeaderFirstSentence renders correctly 1`] = `
<h1
  className="sc-EHOje UZKL"
/>
`;

exports[`StyledHeaderSecondSentence renders correctly 1`] = `
<h2
  className="sc-bZQynM fASZuk"
/>
`;
